src/Bicep.LangServer.IntegrationTests/CompletionTests.cs

```diff
@@ -1014,14 +1013,12 @@ public async Task RequestCompletionsInObjects_AtPositionsWhereNodeShouldNotBeIns
         public async Task RequestCompletionsInArrays_AtPositionsWhereNodeShouldNotBeInserted_ReturnsEmptyCompletions()
         {
             var fileWithCursors = @"
```

Comment:
It would be nice to add a new test method just to verify the end-to-end functionality (request a completion, verify we get expected results back). It doesn't need to be super exhaustive as you've already got some very thorough unit tests, but would just be nice to have something to verify that all of the pieces fit together correctly.

Reply:
Added more tests + fixed the array item fall through logic that wasn't working for some single line array cases when using ternaries or parentheses.